stable/redis-ha/README.md

```diff
@@ -82,55 +82,58 @@ The following table lists the configurable parameters of the Redis chart and the
 | `tolerations`             | Toleration labels for pod assignment                                                                                                                                                                     | `[]`                                                                                       |
 | `hardAntiAffinity`        | Whether the Redis server pods should be forced to run on separate nodes.                                                                                                                                 | `true`                                                                                     |
 | `additionalAffinities`    | Additional affinities to add to the Redis server pods.                                                                                                                                                   | `{}`                                                                                       |
-| `securityContext`    | Security context to be added to the Redis server pods.                                                                                                                                                        | `{runAsUser: 1000, fsGroup: 1000, runAsNonRoot: true}`                                     |
+| `securityContext`         | Security context to be added to the Redis server pods.                                                                                                                                                   | `{runAsUser: 1000, fsGroup: 1000, runAsNonRoot: true}`                                     |
 | `affinity`                | Override all other affinity settings with a string.                                                                                                                                                      | `""`                                                                                       |
+| `persistentVolume.size`          | Size for the volume                                                                                                                                                                               | 10Gi                                                                                       |
+| `persistentVolume.annotations`   | Annotations for the volume                                                                                                                                                                        | `{}`                                                                                       |
+| `persistentVolume.reclaimPolicy` | Method used to reclaim an obsoleted volume. `Delete` or `Retain`                                                                                                                                  | `Retain`                                                                                   |
```

Comment:
Default should be `nil` as you've commented out the section in the file.

Reply:
Retain is already the default inn k8s. But this is cleaner - thanks.